pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -411,9 +421,15 @@ private void handleReleaseEvent(String serviceUnit, ServiceUnitStateData data) {
 
     private void handleSplitEvent(String serviceUnit, ServiceUnitStateData data) {
         if (isTargetBroker(data.broker())) {
-            splitServiceUnit(serviceUnit)
-                    .thenCompose(__ -> tombstoneAsync(serviceUnit))
-                    .whenComplete((__, e) -> log(e, serviceUnit, data, null));
+            splitServiceUnit(serviceUnit, data)
+                    .whenComplete((__, e) -> {
+                        if (e != null) {
+                            // When has exception, change the bundle state back to Splitting -> Owned .
+                            pubAsync(serviceUnit, new ServiceUnitStateData(Owned, data.broker(), data.sourceBroker()));
```

Comment:
I am sorry. I thought about this again, and in fact, this revert `Splitting -> Owned` can cause an inconsistent state if the exception is thrown after the children partitions are updated in the metadata store or children bundles are created.

Let's keep the exception-handling logic simple (do not revert the state here). Currently, the ServiceUnitStateChannel recovery logic will automatically tombstone the ownerships of any in-flight state service units, which I think is enough.